File: 02_Lenguajes-de-programación/10_Go/05_bucles.go
package main

import ("fmt")


func main() {

	// Bucles

	for i := 0; i < len(myArray); i++ {
		fmt.Println(myArray[index])
	} // 1 \n 2 \n 3

	for index, value := range myMap {
		fmt.Println(index, value)
	}
	
}

File: 02_Lenguajes-de-programación/06_PHP/05_bucles.php
<?php

// Ciclos (Bucles)

// - For
for ($index = 0; $index <= 10; $index++) {
    echo $index . "\n";
}

// - Foreach
foreach ($my_array as $my_item) {
    echo $my_item . "\n";
}

// - While

$index = 0;
while ($index <= sizeof($my_array) - 1) {
    echo $my_array[$index] . "\n";
    $index++;
}
